src/processors/processVrmRequest.ts

```diff
@@ -13,7 +13,7 @@ export const processPatchVrmRequest = (currentRecord: TechRecordGet, userDetails
     const updatedNewRecord = setCreatedAuditDetails(newRecord, userDetails.username, userDetails.msOid, new Date().toISOString(), currentRecord.techRecord_statusCode as StatusCode);
     (updatedNewRecord as TechRecordHgv | TechRecordMotorcycle | TechRecordCar | TechRecordPsv).primaryVrm = newVrm.toUpperCase();
     const updatedRecordToArchive = setLastUpdatedAuditDetails(recordToArchive, userDetails.username, userDetails.msOid, new Date().toISOString(), StatusCode.ARCHIVED);
-
+    newRecord.techRecord_reasonForCreation = 'Update VRM - Cherished Transfer';
```

Comment:
should be `updatedNewRecord`

Reply:
That part is fine, there's a `updatedNewRecord` with the audit details and new vrm, that's the one you would want to use. (There's `updatedRecordToArchive` for the archive one)